query_test.go

```diff
@@ -1,398 +1,407 @@
-package gorqlite
+package gorqlite_test
 
 import (
+	"context"
 	"fmt"
 	"testing"
 	"time"
+
+	"github.com/rqlite/gorqlite/v2"
 )
 
 func TestQueryOne(t *testing.T) {
-	var wr WriteResult
-	var qr QueryResult
-	var wResults []WriteResult
-	var qResults []QueryResult
-	var err error
-
-	t.Logf("trying Open")
-	conn, err := Open(testUrl())
-	if err != nil {
-		t.Logf("--> FATAL")
-		t.Fatal()
-	}
-
-	t.Logf("trying WriteOne DROP")
-	wr, err = conn.WriteOne("DROP TABLE IF EXISTS " + testTableName())
```

Comment:
I thought we were gonna keep creating/dropping tables per test

Reply:
Yes, we are. See line 38-48 of this file

Reply:
Ah I missed that, thanks.